app/helpers/react_components/mentoring/session.rb

```diff
@@ -31,7 +31,7 @@ def to_s
               anonymous_mode: discussion&.anonymous_mode?
             ),
             mentor_solution: mentor_solution,
-            exemplar_solution: exercise.exemplar_files.values.first,
+            exemplar_files: SerializeGitFiles.(exercise.exemplar_files),
```

Comment:
@kntsoriano I've inlined the serializing here, as it needs custom rules for this situation (stripping the `.meta`)